package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "hello-microverse",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/richy4dev/Hello-Microverse.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/richy4dev/Hello-Microverse/issues"
+  },
+  "homepage": "https://github.com/richy4dev/Hello-Microverse#readme",
+  "devDependencies": {
+    "hint": "^7.1.1",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
+  }
+}
```

Comment:
## [Optional]
### Branch name
![branch (copy)](https://user-images.githubusercontent.com/93039359/184938111-8b4e1db8-7d6a-4451-bd4d-ee48a949deb4.png)


- [ ] Your branch name is not descriptive enough. You are using **Linter-setup**, you can use  **feature/setup-hello-microverse** or **feature/initialize-hello-microverse** :+1: